config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/BundleArtifacts.java

```diff
@@ -12,11 +12,13 @@
 
     private final Element bundle;
     private final Element deleteBundle;
+    private final Element deleteEnvBundle;
```

Comment:
Since both of them being generated in different tasks, I think we can use `deleteBundle` only. No need for new variable. We are also using `bundle` in the same way.

Reply:
done